source/localizable/tutorial/routes-and-templates.md

```diff
@@ -105,22 +109,21 @@ Since we will be using the `contact` template, the `contact` route does not need
 In `contact.hbs`, we can add the details for contacting our Super Rentals HQ:
 
 ```app/templates/contact.hbs
-<p>Super Rentals Representatives would love to help you choose a destination or answer
-any questions you may have.</p>
-
-<p>Contact us today:</p>
-
-<p>
-  Super Rentals HQ
-  <address>
-    1212 Test Address Avenue<br>
-    Testington, OR 97233
-  </address>
-</p>
-
-<p><a href="tel:503.555.1212">(503)555-1212</a></p>
-
-<p><a href="mailto:superrentalsrep@superrentals.com">superrentalsrep@superrentals.com</a></p>
+<div class="jumbo">
+  <div class="right tomster"></div>
+  <h2>Contact Us</h2>
+  <p>Ember Rentals Representatives would love to help you<br>choose a destination or answer
+    any questions you may have.</p>
+  <p>
+    Ember Rentals HQ
+    <address>
+      1212 Test Address Avenue<br>
+      Testington, OR 97233
+    </address>
+    <a href="tel:503.555.1212">+1 (503) 555-1212</a><br>
+    <a href="mailto:emberrentalsrep@emberrentals.com">superrentalsrep@superrentals.com</a>
```

Comment:
Do we own that domain? Are we deploying the app? I don't like having email links to whoever knows :\